### PR TITLE
Fix deadloop in UpdateSummary.

### PR DESCRIPTION
**Pull request by LiliDeng**
When execute below command directly will enter into dead loop.
```
.\Tools\plink.exe -ssh -t -pw ****** -P 22 username@IP "echo ****** | sudo -S . utils.sh && update_repos && insta
ll_package 'make kernel-devel gcc-c++'"
```
LogErr and UpdateSummary invoked each other.
https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L190
https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L205

### CODE REVIEW

File: Testscripts/Linux/utils.sh [hunk 1]
```diff
@@ -138,12 +138,12 @@ function __SetTestState() {
 		if [ -w "$__LIS_STATE_FILE" ]; then
 			echo "$1" > "$__LIS_STATE_FILE"
 		else
-			LogErr "State file $__LIS_STATE_FILE exists and is a normal file, but is not writable"
+			LogMsg "State file $__LIS_STATE_FILE exists and is a normal file, but is not writable"
```
Comment (LiliDeng) [inline Testscripts/Linux/utils.sh:141]: the following command will make the file executable.

File: Testscripts/Linux/utils.sh [hunk 2]
```diff
@@ -138,12 +138,12 @@ function __SetTestState() {
...
 			chmod u+w "$__LIS_STATE_FILE" && { echo "$1" > "$__LIS_STATE_FILE" && return 0 ; } || LogMsg "Warning: unable to make $__LIS_STATE_FILE writeable"
 			return 1
 		fi
 	else
-		LogErr "State file $__LIS_STATE_FILE either does not exist or is not a regular file. Trying to create it..."
+		LogMsg "State file $__LIS_STATE_FILE either does not exist or is not a regular file. Trying to create it..."
```
Comment (LiliDeng) [inline Testscripts/Linux/utils.sh:146]: following command will touch a state file

**Review by lubaihua33 — APPROVED** [review]

**Comment by juhlee-microsoft** [comment]
Help me understand this PR. Its title said "Fix deadloop in UpdateSummary" but the changes do not seem to corporate.

**Comment by LiliDeng** [comment]
@juhlee-microsoft In https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L205 UpdateSummary function, it will invoke LogErr https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L190, then in LogErr it will invoke UpdateSummary. I use LogMsg like before your change, since they are not error, in following script, it will create file if file not exist etc.

**Review by juhlee-microsoft — APPROVED** [review]

**Comment by juhlee-microsoft** [comment]
> @juhlee-microsoft In https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L205 UpdateSummary function, it will invoke LogErr https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/utils.sh#L190, then in LogErr it will invoke UpdateSummary. I use LogMsg like before your change, since they are not error, in following script, it will create file if file not exist etc.

Got it. thanks

**Pull request merged**